cfme/tests/migration_analytics/test_migration_analytics.py

```diff
@@ -0,0 +1,206 @@
+"""Tests around the Migration Analytics"""
+import pytest
+
+from cfme import test_requirements
+
+pytestmark = [
+    pytest.mark.ignore_stream("5.10"),
+    pytest.mark.tier(1),
+    test_requirements.migration_analytics
+]
+
+
+@pytest.mark.manual
+@pytest.mark.smoke
+def test_enable_migration_analytics():
+    """
+    Polarion:
+        assignee: ghubale
+        initialEstimate: 1/4h
+        startsin: 5.11
+        casecomponent: MigrationAnalytics
+        testSteps:
+            1. Navigate to configuration > Zones > Zone: Default Zone > Server
+            2. Click on Advanced tab
+            3. Update script as below:
+               :prototype:
+                    :migration_analytics:
+                    :enabled: true
+            4. SSH to appliance and restart evmserverd(systemctl restart evmserverd)
+        expectedResults:
+            1.
+            2.
+            3.
+            4. Check in UI. You will have navigation to Migration > Migration Analytics
+    """
+    pass
+
+
+@pytest.mark.manual
+def test_payload_generation_for_basic_data_without_ssa():
+    """
+    Polarion:
+        assignee: ghubale
+        initialEstimate: 1/4h
+        startsin: 5.11
+        casecomponent: MigrationAnalytics
+        setup:
+            1. Add VMWare provider
+        testSteps:
```

Comment:
Don't you need to enable MA before using it? You could add it to setup steps here and for all the tests.

Reply:
Yes Agree :)
But this common and required one. Without this we cant run any MA test case. So didn't add to each test.
Let me know your view :)

Reply:
Agreed. But I have seen everyone adding such prerequisites in the testSteps/setup even if they are common, for e.g. enable CandU or setup SMTP. If you don't want to add this step, that is also fine.